engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/puppet-modules/content-view-puppet-modules.controller.js

```diff
@@ -26,7 +26,10 @@ angular.module('Bastion.content-views').controller('ContentViewPuppetModulesCont
         $scope.versionText = function (module) {
             var version;
             if (module['computed_version']) {
-                version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
+                if (module['uuid'])
+                    version = translate("Currently %s").replace('%s', module['computed_version']);
+                else
+                    version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
```

Comment:
syntactically you are correct, but our style dictates to always use braces.  The reason for this is that it reduces errors.  If someone comes back in and adds a second line they may forget to add curly braces and get unexpected behavior.  It also increases readability (that is a bit of an opinion, but is pretty common).

The CI job failed due to the style checker hitting this error